frontend/src/modules/placement/classes.ts

```diff
@@ -802,6 +808,10 @@ export class SampleClass extends PlacementObject {
     get fromCell() {
         return this.state.fromCell ? this.placement.getCell(this.state.fromCell) : null
     }
+    
+    fromParentContainer() {
```

Comment:
Remove this please if this is not referenced anywhere (yet).

Reply:
DONE